src/schema/objects/metadata.yaml

```diff
@@ -2649,6 +2833,20 @@ SEEGChannelCount:
     Number of SEEG channels.
   type: integer
   minimum: 0
+SampleCoordinateSystem:
+  name: SampleCoordinateSystem
+  display_name: SampleCoordinateSystem
+  description: |
+    Classical screen-based eye tracking data would be `"gaze-on-screen"`,
+    but `"eye-in-head"` or `"gaze-in-world"` are also possible coordinate systems
+    (for example for virtual reality).
```

Comment:
Should this be an enum?

Reply:
Ok I'll go with the second option. Thanks